docs/template.asciidoc

```diff
@@ -0,0 +1,95 @@
+[[page-title]]
+= Page title
+
+////
+To make it easier to scan the TOC, include critical information first. For example, instead of `Backing up your data with snapshot and restore`, use `Snapshot and restore`.
+////
+
+Include a short description. The short description should always include 
+the value prop, answer why users want to use the feature, and how the 
+feature benefits them. 
+
+[partintro]
+--
+
+For easy navigation, include the links to the key sections in a bulleted 
+list. This way, users can click exactly where they want to go instead of 
+scrolling down the page:
+
+* <<task-section-description, Task section title>>
```

Comment:
Should this link to concepts and example as well as tasks?

Reply:
Since the concept section should come right after the intro section, I wouldn't add it to the list of links.